CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+# 2.8.2 (2018-04-05)
+
+- [FIXED] Added special handling for requesting the `_users` database, as it can send `404` errors when a session expires.
```

Comment:
Please move this CHANGES entry under `Unreleased` and remove the `# 2.8.2 (2018-04-05)` line.  Our team will handle releasing the next version.

Reply:
Sure!